content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -163,19 +163,11 @@ kubectl get events --sort-by=.metadata.creationTimestamp
 
 ## Updating Resources
 
-As of version 1.11 `rolling-update` have been deprecated (see [CHANGELOG-1.11.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.11.md)), use `rollout` instead.
-
 ```bash
```

Comment:
Not sure on this change: May be helpful to keep the note about deprecating `rolling-update`?
Could add a single note, As of version 1.11 `kubectl rolling-update` has been deprecated (...).

Use `kubectl rollout` to <update a Deployment>?

Reply:
OK